policy/predicate/title.go

```diff
@@ -28,22 +28,45 @@ type Title struct {
 
 var _ Predicate = Title{}
 
-func (pred Title) Evaluate(ctx context.Context, prctx pull.Context) (bool, string, error) {
+func (pred Title) Evaluate(ctx context.Context, prctx pull.Context) (*common.PredicateResult, error) {
 	title := prctx.Title()
 
+	predicateResult := common.PredicateResult{
+		Satisfied:       false,
+		ValuePhrase:     "title",
```

Comment:
```suggestion
		ValuePhrase:     "titles",
```

Reply:
shouldn't there be only 1 title for each pr?